pandas/core/arrays/interval.py

```diff
@@ -1015,6 +1017,40 @@ def repeat(self, repeats, **kwargs):
         right_repeat = self.right.repeat(repeats, **kwargs)
         return self._shallow_copy(left=left_repeat, right=right_repeat)
 
+    _interval_shared_docs['overlaps'] = """\
+        Check elementwise whether the given input overlaps the intervals in
+        the %(klass)s.
+
+        .. versionadded:: 0.24.0
+
+        Parameters
+        ----------
+        other : Interval-like
```

Comment:
Why Interval-like and not Interval ?

Reply:
Changed.  Originally wrote Interval-like to mean Interval/IntervalIndex/IntervalArray, as there were originally plans to support all three.  Probably best to just say Interval for now since that's the only thing supported, and leave such a modification for when the other two are implemented.